llvm/lib/Target/RISCV/RISCVISelLowering.h

```diff
@@ -334,8 +334,8 @@ enum NodeType : unsigned {
 
 class RISCVTargetLowering : public TargetLowering {
   const RISCVSubtarget &Subtarget;
-  // int *VastartStoreFrameIndex = new int(-1);
-  int *VastartStoreFrameIndex = new int;
+  int *VastartStoreFrameIndex = new int(INT_MAX);
```

Comment:
The name should be sth like `VaStartStoreFrameIndex`, 
also can we use `int VaStartStoreFrameIndex = INT_MAX` instead?

Reply:
You mean commit name? Maybe I should rebase them in the end.
I think use `int VaStartStoreFrameIndex = INT_MAX` is good.

Reply:
`int *VastartStoreFrameIndex` is used in const functions, shouldn't be modified.